Fix field names used by the welcome package metric events (#73)
atom/welcome: docs/events.md

# Events specification

> This document specifies all the data (along with the format) which gets send from the Welcome package to the GitHub analytics pipeline. This document follows the same format and nomenclature than the [Atom Core Events spec](https://github.com/atom/metrics/blob/master/docs/events.md).

Reviewer: ```suggestion
This document specifies all the data (along with the format) which gets sent from the Welcome package to the GitHub analytics pipeline. This document follows the same format and nomenclature as the [Atom Core Events spec](https://github.com/atom/metrics/blob/master/docs/events.md).
```

## Counters

> Currently the Welcome package does not log any counter event.

Reviewer: ```suggestion
Currently the Welcome package does not log any counter events.
```

## Timing events

> Currently the Welcome package does not log any timing event.

Reviewer: ```suggestion
Currently the Welcome package does not log any timing events.
```
